sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,21 +20,69 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      sourceTypeSpecificValues:
+        fieldsToReturn:
+          - external_identifier
+          - tag
+          - group
+          - contact_type
+          - contact_sub_type
+          - last_name
+          - first_name
+          - middle_name
+          - email
+          - source
+          - user_unique_id
+          - organization_name
+          - phone
+          - city
+          - modified_date
+    dataUrl:
+      urlExcludingConfigurableParameters: https://crm.elifesciences.org/crm/sites/all/modules/civicrm/extern/rest.php?entity=Contact&action=get
+      configurableParameters:
+        fromDateParameterName: modified_date
+        defaultStartDate: "2012-01-01 00:00:00"
+        dateFormat: "%Y-%m-%d %H:%M:%S"
+        pageSizeParameterName: limit
+        defaultPageSize: 10000
+        offsetParameterName: offset
+      parametersFromEnv:
+        - parameterName: api_key
```

Comment:
I am surprised the main way of passing the key seem to be as a URL parameter. It will be in the access logs can more easily leak.

Reply:
you are right